client/components/chart/utils.js

```diff
@@ -30,6 +30,25 @@ d3FormatDefaultLocale( {
 	grouping: [ 3 ],
 	currency: [ decodeSymbol( get( wcSettings, 'currency.symbol', '' ) ), '' ],
 } );
+
+/**
+ * Describes `smallestFactor`
+ * @param {number} inputNum - any double or integer
+ * @returns {integer} smallest factor of num
+ */
+export const getFactors = inputNum => {
+	const num = Math.round( inputNum );
```

Comment:
I think we don't need to round this number here. Isn't it always an integer?

Reply:
For this use case, yes. We're expecting an integer. I'll update.